### ISSUE TITLE
misspelled-properties change in 1.11.0 breaks nested properties

### ISSUE THREAD

**Issue by christianrank** [sasstools/sass-lint#1112]
**Used version**: 1.11.0

```scss
@mixin font-test1 {
  font: {
    family: Arial, sans-serif;
    style: normal;
    weight: 300;
  }
}

@mixin font-test2 {
  font: {
    family: Arial, sans-serif;
    style: normal;
    weight: 400;
  }
}
```

results in the following error:

```  13:5  error  Property `weight` appears to be spelled incorrectly  no-misspelled-properties```

It's also weird that it only complains about the second `weight`, if there's only one mixin, it doesn't complain at all.

**Expected**: There shouldn't be an error.

**Comment by DanPurdy** [comment]
Thanks for the report, will look into it now.

**Comment by DanPurdy** [comment]
Fix incoming with the next release.

**Comment by Grawl** [comment]
causing with `.sass` #1290
